### DISCUSSION TITLE
飞布如何在微服务场景下使用

### DISCUSSION THREAD

**Discussion by hefangxuan** [fireboomio/product-manual#75] [Q&A]
假设有 A、B、C三个服务 或者是三个函数 他们可能部署在不同的服务器上
现在想要使用飞布作为网关层管理对外提供接口，（我刚接触到飞布，我的理解这个需求应该是可以实现的）
那么我使用飞布作为网关层的时候，还希望能完全使用低代码特性
或者换个说法，我使用飞布创建的数据模型并且创建了接口，而且也实现了鉴权等。
那我是否可以将某一块功能划分为A、B、C的服务单独部署到不同的服务器

或者说飞布创建的接口服务，怎样做分布式 处理高并发业务场景
新人 + 刚接触到飞布，问题不知是否表述清楚，还望理解 加以帮助 谢谢

**Comment by AnsonCode** [comment]
如果想要应对高并发场景，建议把Fireboom封装为DOCKER镜像，通过k8s调度。
其中Fireboom镜像应该包含全部接口。
